public/data/ankanbag101.json

```diff
@@ -0,0 +1,67 @@
+{
+    "name": "Ankan Bag",
+    "type": "personal",
+    "bio": "An alien👽 trying to make a difference in the tech world🔥",
+    "avatar": "https://github.com/ankanbag101.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/ankanbag101",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/AnkanBag06",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://linkedin.com/in/ankan-bag-74a238222/",
+        "icon": "linkedin"
+      },
+      {
+        "name": "Email me",
+        "url": "mailto:bagankan@gmail.com",
+        "icon": "envelope"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started GitHub account",
+        "date": "October 2021",
+        "icon": "github",
+        "color": "black",
+        "description": "Opened GitHub account for organising all my local codebase",
+        "url": "https://github.com/ankanbag101"
+      },
+      {
+        "title": "Won first prize in web-battle-hackathon",
+        "date": "October 2021",
+        "icon": "dollar",
+        "color": "green",
+        "description": "First time participated on a hackathon and won first prize of 5000 rupees"
+      },
+      {
+        "title": "published first article on Medium",
+        "date": "October 2021",
+        "icon": "dollar",
+        "color": "orange",
+        "description": "Wrote my first 'how to' blog on AWS S3",
+        "url": "https://ankan101.medium.com/how-to-upload-large-files-to-aws-s3-8be5d6da8374"
+      },
+      {
+        "title": "Shortlisted among the top 10 AR Creator",
+        "date": "December 2021",
+        "icon": "dollar",
+        "color": "red",
+        "description": "Won $100 in the Hackathon organized by Reskill and Spark AR on creating AR effects. Was placed in Top 10 out of 2000+ applications."
+      },
+      {
+        "title": "First Contribution to Open-Source",
+        "date": "December 2021",
+        "icon": "github",
+        "color": "grey",
+        "description": "Contributed to Daily.dev readme file"
```

Comment:
```suggestion
        "description": "Contributed to Daily.dev README file"
```

Reply:
Thanks for the suggestion, I will change it.